app/pb_kits/playbook/pb_progress_step/progress_step.rb

```diff
@@ -31,6 +34,13 @@ def variant_class
           "tracker"
         end
       end
+
+      def color_class
+        case color
+        when "info"
+          "info"
+        end
+      end
```

Comment:
👍 Looks like we are setting this up for additions later. Good deal.

Reply:
yup we may incorporate warning or error in the future. For now the color is meant for the tracker variant @thestephenmarshall